python/lsst/afw/cameraGeom/basicCameraFactory.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python2
+#
+# LSST Data Management System
+# Copyright 2014 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division
+import argparse
+import os
+import re
+import shutil
+
+import lsst.utils
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+from lsst.afw.cameraGeom import makeCameraFromCatalogs
+from lsst.afw.cameraGeom import (DetectorConfig, CameraConfig, PUPIL, FOCAL_PLANE, PIXELS,
+                                 SCIENCE, FOCUS, GUIDER, WAVEFRONT)
```

Comment:
Are all these used? GUIDER doesn't seem to be mentioned anywhere other than the documentation.